src/pages/workspace/distanceRates/PolicyDistanceRateDetailsPage.tsx

```diff
@@ -115,55 +126,90 @@ function PolicyDistanceRateDetailsPage({policy, route}: PolicyDistanceRateDetail
                     threeDotsMenuItems={threeDotsMenuItems}
                     threeDotsAnchorPosition={styles.threeDotsPopoverOffset(windowWidth)}
                 />
-                <View style={styles.flexGrow1}>
-                    <OfflineWithFeedback
-                        errors={ErrorUtils.getLatestErrorField(rate ?? {}, 'enabled')}
-                        pendingAction={rate?.pendingFields?.enabled}
-                        errorRowStyles={styles.mh5}
-                        onClose={() => clearErrorFields('enabled')}
-                    >
-                        <View style={[styles.flexRow, styles.justifyContentBetween, styles.p5]}>
-                            <Text>{translate('workspace.distanceRates.enableRate')}</Text>
-                            <Switch
-                                isOn={rate?.enabled ?? false}
-                                onToggle={toggleRate}
-                                accessibilityLabel={translate('workspace.distanceRates.enableRate')}
+                <ScrollView contentContainerStyle={styles.flexGrow1}>
+                    <View>
```

Comment:
Not critical: I think we can safely remove this `<View>`

Reply:
agree.

Reply:
@eVoloshchak Updated.